### DISCUSSION TITLE
Add new themes

### DISCUSSION THREAD

**Discussion by zeroesh** [webreinvent/vaahcms#95] [Help]
Here are fews questions related to adding new themes
1. I want to add new themes but have only html files, is there a way it can be added
2. I have uploaded my themes on google drive will that be fine or i need to find a different url

**Comment by themodernpk** [comment]
1. You can develop a new theme's scaffolding  using `npx vaah cms:theme`, read more at https://docs.vaah.dev/vaahcms/theme/develop-theme.html then you need to convert HTML file into `blade` file which is supported by Laravel.
2. VaahCMS can't read your Google Drive, your theme must present in `<project-root-dir>/VaahCMS/Themes/<your-theme-folder>`

I recommend reading laravel documentation as well as vaahcms.
